src/components/QuestionnaireForm/QuestionnaireForm.jsx

```diff
@@ -7,6 +7,7 @@ import _ from "lodash";
 
 // NOTE: need to append the right FHIR version to have valid profile URL
 var DTRQuestionnaireResponseURL = "http://hl7.org/fhir/us/davinci-dtr/StructureDefinition/dtr-questionnaireresponse-";
+// var DTRQuestionnaireResponse
```

Comment:
This var should not be a class wide variable. It is better to define it in the function which uses it.

Reply:
Hi Yolanda, this is a different link that was coded by someone else. I included my url within the class! I can however fix this link too if you'd like!